acceptable/_service.py

```diff
@@ -161,6 +161,11 @@ def serialize(self):
 
         return api_metadata, locations
 
+    def openapi_31x(self):
+        """Return an object conforming to the OpenAPI 3.1.x specification."""
+
+        return None
```

Comment:
Oh... for a moment forgot this was a wip poc and was looking for the actual implementation! hahah

Reply:
I do expect a call to something like `acceptable.openapi.something()`... maybe?

Reply:
This is WIP but my intention is to merge into main. Would you prefer a feature flag?

Reply:
As mentioned above, maybe we could just pass the output of `.serialize()` to some openapi specific function and leave this class alone.

Reply:
I've moved this out of _service.py, I agree it's neater in the new openapi module.